src/core/stdc/fenv.d

```diff
@@ -125,9 +125,8 @@ else version( DMC_RUNTIME )
     }
     alias fexcept_t = int;
 }
-else version( Windows )
+else version( CRuntime_Microsoft )
```

Comment:
You should also use CRuntime_DigitalMars for the version condition just above this one. DMC_RUNTIME is no longer correct.